quickstart.yml

```diff
@@ -12,6 +12,9 @@
 services:
   hydra:
     image: oryd/hydra:v2.2.0
+    build:
+      context: .
+      dockerfile: .docker/Dockerfile-local-build
```

Comment:
One notable difference that will happen here is due to the decision to continue using non-root, Alpine (_from Docker Hub pull_) will have `500` UID/GID, while Distroless (_from local build_) will have `65532` UID/GID, which will mess with ownership for the db volume if the user runs both variants at some point.

You should document that caveat or correct for it. Instead of the 500 this PR assigns in the Alpine image, it could probably be 65532 to match distroless.

Reply:
That‘s a good point!!

Reply:
changed to 65532 for alpine